python/caffe/pycaffe.py

```diff
@@ -14,6 +14,9 @@
         RMSPropSolver, AdaDeltaSolver, AdamSolver
 import caffe.io
 
+# py3 compliance
+from six import itervalues, next, iteritems
```

Comment:
I guess I'd rather we make our use of `six` explicit by just doing `import six` here... I think that's how we already do it in other files, and `six` is pretty short, and I think it's easier to read that way.

You can also drop the comment, IMO... we all know what `six` does, and anyone who is not familiar with it can find out easily